cmds/core/cp/cp_others.go

```diff
@@ -0,0 +1,12 @@
+// Copyright 2022 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build !plan9
+// +build !plan9
```

Comment:
I think the `+build` line is not necessary as this is legacy behavior for the Go version we're targeting

Reply:
gofmt puts it in automagically, so I think I need to leave it there.